gulpfile.babel.js

```diff
@@ -13,6 +13,7 @@ const buffer = require('vinyl-buffer');
 const eslint = require('gulp-eslint');
 const uglify = require('gulp-uglify');
 const mkdirp = require('mkdirp');
+const envify = require('envify/custom');
```

Comment:
Do you think we need to optimize this build? It only happens for the production example site. I don't expect people do anything that js-tokens can't handle. But the AST portion makes me feel safe